content/docs/command-reference/data/status.md

```diff
@@ -108,6 +117,41 @@ yet, and a file `model.pkl` has been deleted from the workspace. The
 `data/features/` directory is modified, but there is no further details to what
 changed inside. The `--granular` option can provide more information on that.
 
+## Example: Combine with Git status
```

Comment:
Minor: Maybe use a more use-case oriented title? E.g. "Show all changes in the working tree (including Git)"